packages/react-router/docs/README.md

```diff
@@ -5,7 +5,7 @@ Router](https://reactrouter.com)'s documentation, specifically [the
 `react-router` npm package](https://www.npmjs.com/package/react-router).
 
 They were designed to be read online [on our
-website](https://reacttraining.com/react-router/), but that site is temporarily
```

Comment:
Should this be updated, since the API documentation no longer seems to be unavailable?

Reply:
Probably, but we can do that as a separate commit.